<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>dEditUser</class>
  <widget class="QDialog" name="dEditUser" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>414</width>
        <height>246</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>User</string>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <layout class="QVBoxLayout" >
          <item>
            <widget class="QLabel" name="textLabel1" >
              <property name="text" >
                <string>Login</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="lineEdit1" />
          </item>
          <item>
            <widget class="QLabel" name="textLabel2" >
              <property name="text" >
                <string>Password</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="lineEdit2" >
              <property name="echoMode" >
                <enum>QLineEdit::Password</enum>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLabel" name="textLabel3" >
              <property name="text" >
                <string>First name</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="lineEdit3" />
          </item>
          <item>
            <widget class="QLabel" name="textLabel4" >
              <property name="text" >
                <string>Last Name</string>
              </property>
              <property name="wordWrap" >
                <bool>false</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QLineEdit" name="lineEdit4" />
          </item>
        </layout>
      </item>
      <item row="0" column="1" >
        <layout class="QVBoxLayout" >
          <item>
            <widget class="QPushButton" name="pushButton1" >
              <property name="text" >
                <string>Ok</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="pushButton2" >
              <property name="text" >
                <string>Cancel</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer1" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>141</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <includes>
    <include location="local" >auser.h</include>
    <include location="local" >mdtree.h</include>
  </includes>
  <connections>
    <connection>
      <sender>pushButton1</sender>
      <signal>clicked()</signal>
      <receiver>dEditUser</receiver>
      <slot>onOk()</slot>
    </connection>
    <connection>
      <sender>pushButton2</sender>
      <signal>clicked()</signal>
      <receiver>dEditUser</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
